docs/use-rabbitmq.md

```diff
@@ -0,0 +1,30 @@
+---
+title: How to Use RabbitMQ
+---
+<!--
+Copyright (c) 2024 Broadcom. All Rights Reserved. The term "Broadcom" refers
+to Broadcom Inc. and/or its subsidiaries.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the under the Apache License,
+Version 2.0 (the "License”); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# How to Use RabbitMQ
+
+This section targets **developers** creating applications that exchanges
+messages through RabbitMQ.
+
+You might want to start with [Getting Started](/tutorials) if you are new to
+RabbitMQ. These tutorials will guide you on how to use RabbitMQ. The rest of
```

Comment:
rephrasing diffferently: "If you are new to RabbitMQ, you might want to start with [Getting Started](/tutorials).  These tutorials will guide you on how to use RabbitMQ....